### DISCUSSION TITLE
How to run in desktop mode as opposed to mobile

### DISCUSSION THREAD

**Discussion by DH250439** [GoogleChrome/lighthouse-ci#941] [Q&A]
Is there a way to set lighthouse to run in desktop mode in the ci file?

If so, how?

`ci:
  collect:
    # Puppeteer is used to login
    puppeteerScript: puppeteer-script.js
    puppeteerLaunchOptions:
      args:
        - '--no-sandbox'
        - '--headless'
        - '--ignore-certificate-errors'
    numberOfRuns: 1
    settings:
      onlyCategories:
        - accessibility
        - best-practices
        - performance
  assert:
    preset: lighthouse:recommended
    assertions:
      categories:performance:
        - error
        - minScore: .5
      categories:accessibility:
        - error
        - minScore: .85
      categories:best-practices:
        - error
        - minScore: .8
`

![image](https://github.com/GoogleChrome/lighthouse-ci/assets/109621412/54e8d073-05aa-451b-ba01-de59bdf90b04)

**Comment by adamraine** [comment]
https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#desktop-emulation
